test/functional/forall/atomic-ref/tests/test-forall-atomic-ref-other.hpp

```diff
@@ -347,12 +347,23 @@ typename std::enable_if<
          >::type
 testAtomicRefOtherOp(RAJA::RangeSegment seg, T* count, T* list)
 {
+  // Use seg, count, list without changing actual contents to satisfy gcc4.9.3 warnings
```

Comment:
This can also be done by using a with something like the below.
```
// ignore_unused(get, count, list); // silences warnings about unused params
template < typename ... Ts >
void ignore_unused(Ts&&...) { }
```

Reply:
or use RAJA_UNUSED_ARG.
https://github.com/LLNL/RAJA/blob/2a5e34c1d9f928e24c5fa58b90fd14b840c479fa/include/RAJA/util/macros.hpp#L80

Reply:
Thanks, done.